### PR TITLE
CDPT-1062 CAIT: Putting your children first content changes

### PR DESCRIPTION
**Pull request by MeenaModhvadia**
Amended content and links as per ticket.

### CODE REVIEW

File: app/views/pages/homepage.html.erb [hunk 1]
```diff
@@ -48,6 +48,19 @@
           <p class="summary_area__lede-additional govuk-body">Most children will feel more secure if
             they have regular contact with their parents and know where to get extra support if they need it. If they
             understand the situation you should involve them in the decision-making.</p>
+          <p class="summary_area__lede-additional govuk-body">Separating from your partner can be a stressful
```
Comment (vertism) [inline app/views/pages/homepage.html.erb:51]: I think these changes should be in the `involving_children` template

Comment (MeenaModhvadia) [reply app/views/pages/homepage.html.erb:51]: @vertism 
The ticket says to put the text after existing text 'Putting your children first'
which is only on the homepage

Comment (vertism) [reply app/views/pages/homepage.html.erb:51]: @MeenaModhvadia "Putting your children first" is the title of the page - https://helpwithchildarrangements.service.justice.gov.uk/involving-children.

The title and text are duplicated on the homepage, but the ticket is just for this page.

**Comment by MeenaModhvadia** [comment]
Duplicate pr

**Pull request closed without merging**